instagram-app/src/components/PostContainer/PostContainer.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  Input
+} from "reactstrap";
+import pt from 'prop-types';
+import moment from "moment";
+import PostIcons from './PostIcons';
+import CommentSection from "../CommentSection/CommentSection";
+import "./PostContainer.css";
+
+const PostContainer = ({ post }) => {
+  const getDiffDateInWeeks = timestamp => {
+    const now = moment(new Date());
```

Comment:
Great job with this function! 
Just to let you know there is a method in moment library called fromNow() that do everything automatically.